lib/synapse/haproxy.rb

```diff
@@ -743,12 +752,10 @@ def update_backends(watchers)
           rescue StandardError => e
             log.warn "synapse: unknown error writing to socket"
             @restart_required = true
-            return
           else
             unless output == "\n"
```

Comment:
That will result in a log line for every backend that is configured? That seems a little noisy?